content/en/apps/guides/database/_index.md

```diff
@@ -4,4 +4,50 @@ linkTitle: "Database"
 weight: 100
 description: >
   Managing databases used by CHT applications
----
\ No newline at end of file
+---
+
+## CouchDB
+
+The CHT has a range of CouchDB databases for storing different types of data. By default, databases all start with the prefix "medic".
+
+### medic
+
+The main database, used to store all patient and report data. Data access is protected by API to provide protection on a per document basis.
+
+{{% see-also page="core/overview/db-schema" %}}
+
+### medic-sentinel
+
+Stores the `_local/sentinel-meta-data` document which stores the sequence of the last processed change in the `medic` db. This is used so Sentinel can resume from where it left off and process all changes in order.
+
+This database also stores metadata about the documents in the "medic" database, such as when it was received and which Sentinel transitions have executed on this doc. The UUID of the metadata doc is the same as the UUID of the "medic" doc with "-info" appended at the end.
```

Comment:
I'd love to see a snippet here of what an info doc looks like (or link to more info if it exists somewhere else).

```JSON
{
    "_id": "f8cc78d0-31a7-44e8-8073-176adcc0db7b-info",
    "_rev": "2-6e08756f62fa0595d87a3f50777758dc",
    "type": "info",
    "doc_id": "f8cc78d0-31a7-44e8-8073-176adcc0db7b",
    "latest_replication_date": "2018-08-13T22:02:46.699Z",
    "initial_replication_date": "2018-08-13T22:02:13.625Z"
}
```

Reply:
Done.